src/main/java/nl/ordina/jobcrawler/repo/SkillRepository.java

```diff
@@ -4,25 +4,16 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Modifying;
 import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import org.springframework.stereotype.Repository;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.util.Optional;
 import java.util.UUID;
 
 @Repository
+@RepositoryRestResource
 public interface SkillRepository extends JpaRepository<Skill, UUID> {
```

Comment:
I'm not sure if we can use this because it will restrict us with defaults API's that it handles internally. We cannot customised the response that we want and send to the frontend.

Reply:
After some tests I found out that by programming an own endpoint that clashes with this standard API, Spring neglects the standard API. So if we want to add functionality we can still them in a separate controller class. 

This however would mean that we would have to write our own implementation once we need to change anything about the standard API. Therefore, I will revert to the old own implementation after this merge is done that way the frontend can begin building functionality.